src/Core/maps/src/Handlers/MapElement/MapElementHandler.iOS.cs

```diff
@@ -0,0 +1,62 @@
+﻿using MapKit;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Handlers;
+using Microsoft.Maui.Platform;
+
+namespace Microsoft.Maui.Maps.Handlers
+{
+	public partial class MapElementHandler : ElementHandler<IMapElement, MKOverlayRenderer>
+	{
+		protected override MKOverlayRenderer CreatePlatformElement()
+		{
+			if (VirtualView.MapElementId != null)
+			{
+				if (VirtualView is IGeoPathMapElement)
```

Comment:
should this structure instead check the `MapElementId` types? 

```C#
		protected override MKOverlayRenderer CreatePlatformElement()
		{
			if (VirtualView.MapElementId != null)
			{
				var mapElementId = VirtualView.MapElementId;

				if (mapElementId is MKPolygon mKPolygon)
					return new MKPolygonRenderer(mKPolygon);

				if (mapElementId is MKPolyline line)
					return new MKPolylineRenderer(line);

				if (mapElementId is MKCircle circle)
					return new MKCircleRenderer(circle);
			}

			return new MKOverlayRenderer();
		}
```

Reply:
Yeah, that's easier to read maybe. Fixed.